live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/config/ActiveMQProperties.java

```diff
@@ -0,0 +1,20 @@
+package org.alfresco.hxi_connector.live_ingester.routes.config;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Getter
+@Setter
+@ToString
+@Validated
+@ConfigurationProperties(prefix = "alfresco.ingester.messaging")
+public class ActiveMQProperties
```

Comment:
and this file is missing the copyright header completely.

Reply:
Probably the class shouldn't mention ActiveMQ - it's `MessageQueueProperties` or something.

Reply:
added